src/index.js

```diff
@@ -1 +1,12 @@
 import './style.css';
+import addNewScore from './modules/addNewScore';
+// import getScore from './modules/getScore';
```

Comment:
- [ ] kindly consider removing all the commented lines of codes to make your code base look clean